### PR TITLE
[DOCS] Contributing E2E Test

### LINKED ISSUE #11230: [Docs] Step to set up environment variables missing in the docs for contributing to end-to-end playwright tests
### Current State

The Meshery Docs Page for [Contributing to Meshery's End-to-End Tests using Playwright](https://docs.meshery.io/project/contributing/contributing-ui-tests) does not include the step to set the required environment variables before running the tests.

The environment variables required for the tests to run can be found in [this](https://github.com/meshery/meshery/blob/master/ui/tests/e2e/env.js) file. Presently, we compulsorily need these three variables to be set for the tests to run properly: 

- `REMOTE_PROVIDER_USER_EMAIL`: The user's layer5 account email 
- `REMOTE_PROVIDER_USER_PASSWORD` : The user's layer5 account password
- `PROVIDER_TOKEN`: A provider token for the user, that can be generated from [here](https://meshery.layer5.io/security/tokens)

For running the tests locally, one can use their own layer5 account credentials in the environment variables.

### Desired State

Add a step in the Meshery Docs page for contributing to end-to-end playwright tests, for setting required environment variables in the terminal (temporarily or permanently) before running the tests locally.

For Linux based environments, they can be temporarily set using the `export` command:
```bash
export REMOTE_PROVIDER_USER_EMAIL=<layer5 account email>
export REMOTE_PROVIDER_USER_PASSWORD=<layer5 account password>
export PROVIDER_TOKEN=<provider token>
```
Or, they can be permanently set in the `~/.bashrc` file

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by jerensl**
**Notes for Reviewers**

Documentation for Contributing to E2E Test with Playwright

This PR fixes #11230 

Staging:
https://66bf4609ac8fdb1ba5807b1f--meshery-docs-preview.netlify.app/project/contributing/contributing-ui-tests

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 99d3ac629c56bffe47acf707e71043a3bcde30a9 at: https://66bb7080be029a47eabee379--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
Thank you for using the do-not-merge label to help communicate review readiness.

**Comment by Ashparshp** [comment]
@jerensl let's discuss this PR and get some feedback during the Meshery dev meeting call, if you would. Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?usp=drivesdk).

**Comment by l5io** [comment]
🚀 Preview for commit 450ecf38d5c2e8b173f3f8858e07fe267c99571d at: https://66bc91feab6023820c659ab3--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-ui-tests.md [hunk 1]
```diff
@@ -9,40 +9,121 @@ category: contributing
...
+To automate functional integration and end-to-end testing Meshery uses [Playwright](https://playwright.dev/) as one of the tools to automate browser testing. End-to-end tests run with each pull request to ensure that the changes do not break the existing functionality.
 
-## Executing Tests
+## Prerequisites:
 
-Clone the `meshery/meshery` repo. Tests are written in JavaScript. Navigate to the [/ui/tests/e2e/](https://github.com/meshery/meshery/tree/master/ui/tests/e2e) directory.
+Before diving into Meshery's testing environment, certain prerequisites are necessary:
 
-Install the dependencies by running the following command:
+- A verified Layer5 cloud account.
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-ui-tests.md:18]: We should be able to support use of an API token from the Local Provider as well, right?

File: Makefile [hunk 2]
```diff
@@ -81,7 +81,7 @@ docker-testing-env:
 #-----------------------------------------------------------------------------
 # Meshery Server Native Builds
 #-----------------------------------------------------------------------------
-.PHONY: server wrk2-setup nighthawk-setup server-local server-skip-compgen server-no-content golangci proto-build error
+.PHONY: server wrk2-setup nighthawk-setup server-local server-skip-compgen server-no-content golangci proto-build error build-server server-binary
```
Comment (leecalcote) [inline Makefile:84]: The changes in this file look good. 👍

File: install/docker/testing/.dockerignore
Comment (leecalcote) [inline install/docker/testing/.dockerignore:1]: 👍

File: install/docker/testing/Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM golang:1.21 as meshery-server
+FROM golang:1.21.0 as meshery-server
```
Comment (leecalcote) [inline install/docker/testing/Dockerfile:1]: I believe that we're safe to use "1.21.8", in case you'd like to bump this up.

**Comment by l5io** [comment]
🚀 Preview for commit 23228fefb8f927e5e54228e3792e1eaa063020a2 at: https://66be17ffa001ab4c81801a1c--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-ui-tests.md (hunk 1 above)
Comment (jerensl) [reply docs/pages/project/contributing/contributing-ui-tests.md]: How about this?
image

**Comment by l5io** [comment]
🚀 Preview for commit 5ed66f08838d7d8b5688bb6c5a02c090a993a830 at: https://66bf4609ac8fdb1ba5807b1f--meshery-docs-preview.netlify.app

**Review by leecalcote — APPROVED** [review]
LGTM

**Pull request merged**